site/book/01-getting-started/02-quickstart.md

```diff
@@ -107,3 +151,5 @@ Now, let's delve into the details.
 
 TODO: Add the output from running all the commands.
 TODO: Version the nginx example using a tag.
+
+[path expressions]: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/search-replace#matchers
```

Comment:
Remove this reference for couple fo reasons:
1. Details of search and replace is outside the scope of a Quickstart
2. If we were to reference it, it should be the docs site not to GitHub

Reply:
removed.